app/src/main/java/com/mobile/tandil/javabaseproject/utils/Constants.java

```diff
@@ -0,0 +1,15 @@
+package com.mobile.tandil.javabaseproject.utils;
+
+public class Constants {
+
+    public static final int DEFAULT_PARKING = -1;
+    public static final int EMPTY_PARKING = 0;
+    public static final String LISTENER_SET_PARKING_KEY = "LISTENER_KEY";
+    public static final String PARKING_DIALOG_FRAGMENT = "PARKING_DIALOG_FRAGMENT";
+    public static final String DATE_FORMAT = "dd-MMM-yyyy a hh:mm";
+    public static final String LISTENER_DATE_PICKER_KEY = "LISTENER_DATE_PICKER_KEY";
+    public static final String DATE_PICKER_DIALOG_FRAGMENT = "DATE_PICKER_DIALOG_FRAGMENT";
+    public static final String DATE_SELECTOR_BOOLEAN = "DATE_SELECTOR_BOOLEAN";
+    public static final String EMPTY_STRING = "EMPTY_STRING";
```

Comment:
this should be "" instead of "EMPTY_STRING"

Reply:
Done